abnf/odata-abnf-testcases.yaml

```diff
@@ -1812,19 +1812,19 @@ TestCases:
     Input: endswith(["Fred","George","Ron"],["George","Ron"])
 
   - Name: 5.1.1.5.3 indexof
-    Rule: boolCommonExpr
+    Rule: commonExpr
     Input: indexof(CompanyName,'lfreds')
 
   - Name: 5.1.1.5.4 indexof
-    Rule: boolCommonExpr
+    Rule: commonExpr
```

Comment:
But this and the next two *are* `boolCommonExpr`.

Reply:
indexof returns a number